renderer/components/audio-player/volume-slider.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useCallback, useRef, useState } from 'react';
+import { Flex, Box } from 'rebass';
+import { Slider } from '@rebass/forms'
+import throttle from 'lodash.throttle';
+import { ImVolumeHigh, ImVolumeMedium, ImVolumeLow, ImVolumeMute2  } from "react-icons/im";
+
+const VolumeSlider = ({audio}) => {
+    const input = useRef<HTMLInputElement>();
+    const Icon = iconDet => {
+        if (volume > .67) {
+            return <ImVolumeHigh size = "25px"/>;
+        }
+        else if (volume > .33) {
+            return <ImVolumeMedium size = "25px"/>;
+        }
+        else if (volume > 0) {
+            return <ImVolumeLow size = "25px" />;
+        }
+        else {
+            return <ImVolumeMute2 size = "25px" />;
+        }
+    }
+
+    const [volume, setVolume] = useState<number>(1)
```

Comment:
Since `Icon` uses `volume` it makes it easier to understand if `volume` is defined first. Also note that there is no need for `else` in this function since every branch returns.
```suggestion
    const [volume, setVolume] = useState<number>(1);

    const Icon = () => {
        if (volume > .67) {
            return <ImVolumeHigh size = "25px"/>;
        }
        if (volume > .33) {
            return <ImVolumeMedium size = "25px"/>;
        }
        if (volume > 0) {
            return <ImVolumeLow size = "25px" />;
        }
        return <ImVolumeMute2 size = "25px" />;
    }
```

Better yet, `Icon` could take `volume` as a prop and then be defined outside of `VolumeSlider`. This would help performance because currently `Icon` is redefined every render. That would require a bit more changes though so we can leave that for later.